cli/lib/kontena/cli/nodes/ssh_command.rb

```diff
@@ -3,9 +3,9 @@ class SshCommand < Kontena::Command
     include Kontena::Cli::Common
     include Kontena::Cli::GridOptions
 
-    parameter "[NODE_ID]", "SSH to Grid node, or --any"
+    parameter "[NODE_ID]", "SSH to Grid node. Use --any to connect to the first available node"
```

Comment:
Hmm, how does it work with commands anyway? Doesn't the first command get recognized as the node name when using --any?